.github/workflows/release-pypi.yml

```diff
@@ -18,6 +18,21 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - uses: cucumber/action-publish-pypi@v3.0.0
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v5
         with:
-          working-directory: "python"
+          python-version: "3.13"
+
+      - name: Show Python version
+        run: python --version
+
+      - name: Build package
+        run: |
+          python -m pip install build twine
+          python -m build
+          twine check --strict dist/*
```

Comment:
This block is not included in `.github/workflows/test-python.yml`. If these commands fail , we will not know until the build is ran. Please also add these commands there.